package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@unicsmcr/hs_auth_api_ts",
+  "name": "hs_auth_api_ts",
```

Comment:
For the package to be published to the `@unicsmcr` namespace it needs the name to start with `@unicsmcr`.

From the [npm docs](https://docs.npmjs.com/creating-and-publishing-an-org-scoped-package)
```
To verify the package is using your Org scope, in a text editor, open the package’s package.json file and check that the name is @your_org_name/<pkg_name>, replacing your_org_name with the name of your Org.
```

Reply:
Also can we rename the package to `hs_auth_client_ts`?